
Fix build errors like
| ../../v8/src/base/ieee754.cc:1:0: error: switch -mcpu=cortex-a7 conflicts with -march=armv7-a switch [-Werror]

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: chromium-52.0.2743.76/v8/build/toolchain.gypi
===================================================================
--- chromium-52.0.2743.76/v8/build/toolchain.gypi
+++ chromium-52.0.2743.76/v8/build/toolchain.gypi
@@ -201,9 +201,6 @@
               ['v8_target_arch==target_arch', {
                 # Target built with an Arm CXX compiler.
                 'conditions': [
-                  [ 'arm_version==7', {
-                    'cflags': ['-march=armv7-a',],
-                  }],
                   [ 'arm_version==7 or arm_version=="default"', {
                     'conditions': [
                       [ 'arm_fpu!="default"', {
